simple-post-view/requirements.txt

```diff
@@ -0,0 +1 @@
+boto3
```

Comment:
Put into `dev_requirements.txt`. Not required for deployment, so skip to reduce package size